### PR TITLE
Edits from HD and KR

### PR DESCRIPTION
**Pull request by steeleb**
Incorporated suggestions from email chain.

- links open in new tab
- added a few 'important' flags
- added clarity that the only thing that happens the first time you press run is buttons show up
- edited order of tile-dates in validation so the hardest one (and prettiest one) shows up last

I did not change the how to copy the script. I don't think going to the raw file is intuitive. 

Also, dropped the index_GROD files.

### CODE REVIEW

File: docs/index.Rmd [hunk 1]
```diff
@@ -181,14 +183,18 @@ The purpose of this step is to walk you through how to setup a script in GEE tha
...
-9.  Verify that you are in the script that you just saved, then hit 'Run' at the center top of your screen.
+9.  Verify that you are in the script that you just saved, then hit 'Run' at the center top of your screen. It will seem like nothing has happened, but you'll see 'buttons' at the bottom of the map area appear.
 
 <center>![](www/images/run.png){width="350px"}</center>
 
   
 
 Now that you have the script running, you are ready to add labels. Head to the next section, `How to label: Part 2`!
 
+**IMPORTANT**: If you ever get to this point with a new mission-date and there are already labeled pins on your map, STOP, and start from the top of `How to label: Part 1` again. Your map should not have any pins on it!
```
Comment (lindsayplatt) [inline docs/index.Rmd:194]: 💯

**Review by lindsayplatt — APPROVED** [review]
Thanks for implementing these suggestions!

**Comment by steeleb** [comment]
Oh poop, I forgot to merge this before I made some edits. @lindsayplatt, will you look at the edits I made to the eePlumB scripts?

I realized it's hella easier for me to extract band data at the same time that folks extract lat/lon info, so I added that in. Also, put thing that will cause errors in the code if users don't actually fill out the init/date/mission fields.

**Comment by steeleb** [comment]
And so far, I'm the only one who has labeled things, I think, so we aren't missing any data by deploying this edit.

**Review by lindsayplatt — APPROVED** [review]
Looks good! 

Did Hilary or Kait make it past the validation part? They are the only ones who may have labeled already.

**Comment by steeleb** [comment]
Negative.

**Comment by lindsayplatt** [comment]
I will let you actually hit that merge button!

**Comment by lindsayplatt** [comment]
Merging because we need these changes!

**Pull request merged**